content/docs/gke/deploy/deploy-ui.md

```diff
@@ -51,4 +47,60 @@ takes longer than expected, try accessing the above URI anyway.
   [upgrade or reinstall a Kubeflow Pipelines 
   deployment](/docs/pipelines/upgrade/).
 * [Troubleshoot](/docs/gke/troubleshooting-gke) any issues you may
-  find.
\ No newline at end of file
+  find.
+  
+  <!--Sub-sections, these will be not be part of this page  -->
+## Login with GCP Iap {:#Login_with_GCP_IAPSection}
+If you select this option, you see the form:
+<img src="/docs/images/Login with GCP IAP option.png" 
+  alt="Login with GCP IAP option"
+  class="mt-3 mb-3 border border-info rounded">
+- If you haven’t already done so, create an [OAuth Client](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/) and enter the client ID and secret in the field.
+- The default GKE zone is us-central-1a. Use the dropdown to select the zone you want.
+- The Kubeflow version cannot be changed currently.
+
+Note: The three buttons at the bottom of the form are:
+- **Create Deployment**: Click this to deploy your Kubeflow project.
+- **Kubeflow Service Endpoint**: <!-- need info on this--> 
+- **View YAML**: Click this to display the deployment YAML details in a new popup box, as shown below:
+```
+{"appAddress":"https://deploy.kubeflow.cloud","defaultApp":{"components":[{"name":"ambassador","prototype":"ambassador"},{"name":"argo","prototype":"argo"},{"name":"centraldashboard","prototype":"centraldashboard"},{"name":"cert-manager","prototype":"cert-manager"},{"name":"cloud-endpoints","prototype":"cloud-endpoints"},{"name":"gcp-credentials-admission-webhook","prototype":"gcp-credentials-admission-webhook"},{"name":"gpu-driver","prototype":"gpu-driver"},{"name":"iap-ingress","prototype":"iap-ingress"},{"name":"jupyter-web-app","prototype":"jupyter-web-app"},{"name":"katib","prototype":"katib"},{"name":"metacontroller","prototype":"metacontroller"},{"name":"notebook-controller","prototype":"notebook-controller"},{"name":"pipeline","prototype":"pipeline"},{"name":"pytorch-operator","prototype":"pytorch-operator"},{"name":"tf-job-operator","prototype":"tf-job-operator"},{"name":"spartakus","prototype":"spartakus"}],"parameters":[{"component":"cloud-endpoints","name":"secretName","value":"admin-gcp-sa"},{"component":"cert-manager","name":"acmeEmail","value":"user@email.com"},{"component":"iap-ingress","name":"ipName","value":"kubeflow-ip"},{"component":"iap-ingress","name":"hostname","value":"kubeflow.endpoints..cloud.goog"},{"component":"ambassador","name":"platform","value":"gke"},{"component":"spartakus","name":"usageId","value":"73733067"},{"component":"spartakus","name":"reportUsage","value":"true"}],"registries":[{"name":"kubeflow","repo":"https://github.com/kubeflow/kubeflow","version":"v0.5.0","path":"kubeflow"}]}}
```

Comment:
Pretty-print this YAML, for easier reading.

Reply:
done and changed email to a generic naming - to remove my PII